css/properties/clip-path.json

```diff
@@ -4,6 +4,11 @@
       "clip-path": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/clip-path",
+          "spec_url": [
+            "https://drafts.fxtf.org/css-masking/#the-clip-path",
+            "https://www.w3.org/TR/SVG11/masking.html#ClipPathProperty",
```

Comment:
The SVG spec looks dated to me. Is it still relevant?

Reply:
Let’s drop it. It’s not in the SVG2 spec because it’s among the used-to-be-SVG parts that were moved into the FXTF specs.

(pushed e228762 to drop it)